main.go

```diff
@@ -84,6 +87,81 @@ func buildUI() fyne.CanvasObject {
 		}),
 		widget.NewToolbarAction(theme.MailForwardIcon(), func() {
 			code := fmt.Sprintf("%#v", overlay)
+
+			layoutReplace := regexp.MustCompile(`(layout.[a-zA-Z]+)`)
+			code = layoutReplace.ReplaceAllString(code, "${1}Layout") // ToDo: should remove this line once the right layout is picked
+
+			funcReplace := regexp.MustCompile(`\(func\(\)\)\([a-zA-Z0-9]*\)`)
+			code = funcReplace.ReplaceAllString(code, "func(){fmt.Println(\"Hello there\")}")
+
+			areSimplePropsPresent := regexp.MustCompile(`[{ ]+[a-z][a-zA-Z]*:`)
```

Comment:
I can't quite tell what all of these regex do.
They are not named in a way that would hint at the purpose either.
I think all this code should not be in a button callback but in separate methods, something that has good tests - that would show their purpose I think.